packages/Manager/src/main/resources/db/migration/V0044__water_allocation_by_area_add_ruamahanga.sql

```diff
@@ -0,0 +1,31 @@
+CREATE
+OR REPLACE VIEW water_allocations_by_area (area_id, allocation_amount) AS
+SELECT
+  area_id,
+  sum(allocation_plan) as allocation_amount
```

Comment:
i had to change sum(allocation) to sum(allocation_plan) here because 'allocation' is no longer a field. This implies that this view was not working in production. i still have a question mark here because the app was working in prod.

Reply:
of course. thanks!